### ISSUE TITLE
Digital Literacy and Security for the Visually Impaired in Paraguay (TEDIC - OL7 Project - Case Study Culture Track)

### ISSUE THREAD

**Issue by pomodoro24** [MozillaFestival/open-leaders-7#28]
Project Lead: belengimenezcic

Mentor: Natalie García

Welcome to OL7, Cohort E! This issue will be used to track your project and progress during the program. Please use this checklist over the next few weeks as you start Open Leadership Training :tada:.

***

Before Week 1 (Jan 31): Your first mentorship call
- [x] Complete the OLF self-assessment ([online](https://mzl.la/olf-online-assessment), [printable](https://mzl.la/olf-self-assessment)). If you're a group, each teammate should complete this assessment individually. This is here to help you set your own personal goals during the program. No need to share your results, but be ready to share your thoughts with your mentor.
- [x] Make sure you know when and how you'll be meeting with your mentor.

Before Week 2 (Feb 7): First Cohort Call (Open by Design)
- [x] Complete Assignment ['Write Your Vision or Mission Statement'](https://mozilla.github.io/open-leadership-training-series/articles/introduction-to-open-leadership/stating-your-project-vision/). Leave a comment on this issue with your draft vision statement & be ready to share this on the call.
- [x] Watch ['Mozilla & Working Open'](https://youtu.be/quKdaqlR_9w).
- [x] Check the [Syllabus](https://mzl.la/ol7-culture-track-syllabus) for notes and connection info for all the cohort calls.

Before Week 3 (Feb 14): Mentorship call
- [x] Look up two other projects and comment on their issues with feedback on their vision statement.
- [x] Complete your Open Canvas ([instructions](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/), [canvas](https://goo.gl/to6PYn)). Comment on this issue with a link to your canvas.
- [x] [Start your Roadmap](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/). Comment on this issue with your draft Roadmap.

Before Week 4 (Feb 21): Cohort Call (Build for Understanding)
- [x] Look up two other projects and comment on their issues with feedback on their open canvas.
- [x] [Pick an open license](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/sharing-your-work-in-the-open/) for the work you're doing during the program.
- [x] Use your canvas to [start writing a README](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), or landing page, for your project. Link to your README in a comment on this issue.

Week 5 and more
- [x] [Submit your project to Mozilla Pulse](https://www.mozillapulse.org/add), add the tag `#mozsprint`

This issue is here to help you keep track of work as you start Open Leaders. Please refer to the [OL7 Culture Track Syllabus](https://mzl.la/ol7-culture-track-syllabus) for more detailed weekly notes and assignments past week 4.

**Comment by pomodoro24** [comment]
Draft vision statement:
 I’m working with an organization for individuals with visual impairment to teach Digital Literacy and Security workshops to young students so that they can use and create with technology in an agile and safe manner. I’m working open because there is a value in implementing open projects to communities that would not be able to acquire privative software or materials, and because it is more prone to adjust and improve with the contribution of the open community as well.

**Comment by pomodoro24** [comment]
The link to my Open Canvas: https://docs.google.com/presentation/d/1Uzu-mSiCDbAadsMZlcwyfzCKHJRDPajIpOK7_iWxvbw/edit?usp=sharing

**Comment by pomodoro24** [comment]
My draft Roadmap: 

1. Doing more research in depth about open source usability tools and web literacy programs oriented towards the visually impaired (Start: Mid February/End: March)

2. Creating a space that will serve as a repository for what is found and useful (Start: Mid March/ End: ?)

3. Invite others to contribute to the the repository (Start: April/ End: ?)

4. Start with the design of a digital security and web litercy manual specifically for the visually impaired community (Start: Mid April/ End: ?)

**Comment by pomodoro24** [comment]
Open license chosen for this project: CC-BY-SA : [Creative Commons Attribution-ShareAlike 4.0 International License.](https://creativecommons.org/licenses/by-sa/4.0/)

**Comment by msoka86** [comment]
> My draft Roadmap:
> 
> 1. Doing more research in depth about open source usability tools and web literacy programs oriented towards the visually impaired (Start: Mid February/End: March)
> 2. Creating a space that will serve as a repository for what is found and useful (Start: Mid March/ End: ?)
> 3. Invite others to contribute to the the repository (Start: April/ End: ?)
> 4. Start with the design of a digital security and web litercy manual specifically for the visually impaired community (Start: Mid April/ End: ?)

A reasonable and clear roadmap, are you planning to cover other countries from SA in the future?

**Comment by pomodoro24** [comment]
> > My draft Roadmap:
> > 
> > 1. Doing more research in depth about open source usability tools and web literacy programs oriented towards the visually impaired (Start: Mid February/End: March)
> > 2. Creating a space that will serve as a repository for what is found and useful (Start: Mid March/ End: ?)
> > 3. Invite others to contribute to the the repository (Start: April/ End: ?)
> > 4. Start with the design of a digital security and web litercy manual specifically for the visually impaired community (Start: Mid April/ End: ?)
> 
> A reasonable and clear roadmap, are you planning to cover other countries from SA in the future?
 Yes I am! I would like to scale this to many more regions if possible :)
